articles/personal_growth.md

```diff
@@ -0,0 +1,33 @@
+---
+authorName: Sabrina Lange
+authorGithubUsername: sweinzie 
+issue: 97
+title: Personal Growth - Why every Project Should Have Gardeners
```

Comment:
Titelalternativen:
- How  to Foster Personal Growth in Projects 
- What the Projects Gardener Can Do For Your Personal Growth
- Why every Project Should Have Gardeners